docs/LICENSE_OF_DEPENDENCIES.md

```diff
@@ -8,6 +8,7 @@ following works:
 - collectd.org [ISC License](https://github.com/collectd/go-collectd/blob/master/LICENSE)
 - dario.cat/mergo [BSD 3-Clause "New" or "Revised" License](https://github.com/imdario/mergo/blob/master/LICENSE)
 - filippo.io/edwards25519 [BSD 3-Clause "New" or "Revised" License](https://github.com/FiloSottile/edwards25519/blob/main/LICENSE)
+- github.com/99designs/go-keychain [MIT License](https://github.com/99designs/go-keychain/blob/master/LICENSE)
```

Comment:
Could you remove these changes? They don't seem to be needed and are causing a linter test failure

Reply:
I am not sure what to do as it's working blind for me, if I remove those my local checks fail:
```
 veprimk@VEPRIMK-M ~/Desktop/DevOps/Projects/telegraf   add_base64_ieee_float32_converter ±  make check-deps
./scripts/check-deps.sh
darwin/amd64
darwin/arm64
freebsd/386
freebsd/amd64
freebsd/arm
freebsd/arm64
freebsd/riscv64
linux/386
linux/amd64
linux/arm
linux/arm64
linux/loong64
linux/mips
linux/mips64
linux/mips64le
linux/mipsle
linux/ppc64
linux/ppc64le
linux/riscv64
linux/s390x
windows/386
windows/amd64
windows/arm
windows/arm64
 veprimk@VEPRIMK-M
 ```
I will remove those two changes and then will try again.

Reply:
@DStrand1—Thank you for your feedback. I had to add those as they were recommended by the `make check-deps` when executed locally before the git commit. I have removed them as per your recommendations, and now it looks like it works; however, locally, it fails for me.